apps/microtvm/ethosu/requirements.txt

```diff
@@ -0,0 +1,22 @@
+attrs>=21.2.0
```

Comment:
Can you export this file with hashes to ensure integrity of the packages?

Reply:
could you comment where you get the list of packages from? e.g. how would i rebuild this if i change any TVM python deps?

Reply:
@Mousius I've changed this as suggested.